client/js/components/user/orgaDataEntry/OrganisationCompetence.vue

```diff
@@ -0,0 +1,39 @@
+<template>
+  <fieldset
+    v-if="hasPermission('field_organisation_competence')"
+    id="organisationCompetence"
+    class="w-3/4">
+    <legend class="font-size-large weight--normal u-mb-0_75">
```

Comment:
nitpick: Can we use tailwind classes here?

Reply:
I switched them where it was possible [785572d](https://github.com/demos-europe/demosplan-core/pull/4237/commits/785572dcbbcbd10c664361ed3411ab48bbd8de84)